Tasks/app-store-release/app-store-release.ts

```diff
@@ -134,15 +155,18 @@ async function run() {
         tl.debug('Checking for ruby install...');
         tl.which('ruby', true);
 
-        //Whenever a specific version of fastlane is requested, we're going to uninstall all installed
-        //versions of fastlane beforehand.  Note that this doesn't uninstall dependencies of fastlane.
+        //Whenever a specific version of fastlane is requested, we're going to uninstall any installed
+        //versions of fastlane beforehand if exist.  Note that this doesn't uninstall dependencies of fastlane.
         if (installFastlane && fastlaneVersionToInstall) {
-            let gemRunner: ToolRunner = tl.tool(tl.which('gem', true));
-            gemRunner.arg(['uninstall', 'fastlane']);
-            tl.debug(`Uninstalling all fastlane versions...`);
-            gemRunner.arg(['-a', '-I']);  //uninstall all versions
-            await gemRunner.exec();
+            let fastlaneInstalled: boolean = await isFastlaneInstalled();
```

Comment:
`let` to `const`

Reply:
if you use `execSync` in `isFastlaneInstalled` it doesn't have to be async anymore and you can just `if (isFastlaneInstalled())`